softwareComponents/isoreconfig/include/isoreconfig/geometry.hpp

```diff
@@ -13,25 +13,54 @@ constexpr double ERROR_MARGIN = 1.0 / 1000000;
  */
 using Point = arma::vec3;
 /**
- * @brief "Cloud" of points
+ * @brief Container of points
  */
 using Cloud = std::vector< Point >;
 /**
  * @brief "Position" (4x4 matrix)
  */
 using Matrix = arma::mat44;
+/**
+ * @brief Container of positions
+ */
+using Positions = std::vector< Matrix >;
```

Comment:
Zavádíš typedef, který ale nepoužíváš konzistentně. Je to záměr? Chceš rozlišovat mezi `Points` a `vector< Matrix >`?

Reply:
O záměr se nejedná; zapomněl jsem nahradit všechny výskyty přejmenovaného typu novým názvem.